Program/Sound

```diff
@@ -0,0 +1,121 @@
+import random
+
+frq = 12500
+amp = 10
+chan = 1
+hz = 22050
+numsamples = 10
+S = sineWave(600, 2000, 5)
+D = sineWave(440, 2000, 5)
+  
+def sineWave(freq, amplitude, length): 
+  # Create a blank sound 
```

Comment:
This should be a [Doc-String](https://www.python.org/dev/peps/pep-0257/). Also it needs to be more descriptive.